ContentAwareGenerator.php

```diff
@@ -155,9 +155,9 @@ protected function getBestLocaleRoute(SymfonyRoute $route, $parameters)
      * @param array $parameters which should contain a content field containing
      *                          a RouteReferrersReadInterface object
      *
-     * @return SymfonyRoute the route instance
-     *
      * @throws RouteNotFoundException if no route can be determined
+     *
+     * @return SymfonyRoute the route instance
```

Comment:
that seems wrong to me as well. i think just about every code i used had the param / return / throws order.